lib/versioning/VersionID.ts

```diff
@@ -85,6 +89,23 @@ function wait(span: number) {
     }
 }
 
+export function getVersionIdSeed(): string {
```

Comment:
is this logic used in metadata as well?
seeng how this was handled in mongo, I guess either this is done specifically with mongo backend, or the code is duplicated in metadata:
* in the first case, it means this should probably stay in MongoClientInterface;
* in that second case, we should probably create a ticket to clean this up in metadata and use the new code.